Add support for multi language (#4)
terrestris/geoserver-introduction-ws: src/en/README.md

![](./assets/geoserver-logo.png)

# Introduction to GeoServer

> Welcome to the **Introduction of GeoServer - Workshop**.

Author: ```suggestion
Welcome to the **Introduction to GeoServer - Workshop**.
```

> This workshop was developed for use on [OSGeo-Live {{ book.osGeoLiveVersion }} DVD](https://live.osgeo.org) and is intended to give you a comprehensive overview of GeoServer as a web mapping solution. <!---Hinweis auf DVD? -->

Author: ```suggestion
This workshop is developed to be used within the [OSGeo-Live {{ book.osGeoLiveVersion }}](https://live.osgeo.org) environment and is intended to give you a comprehensive overview of GeoServer as a web mapping solution.
```

Note about the DVD is not needed here.

> **info**
>

> > The workshop can be downloaded [as PDF here]({{ book.workshopPdfUrl }}).  <!---PDF auf englisch? -->

Author: To fix the download links please proceed as follows: 
* Replace the variable `workshopPdfUrl` (in book.json) with `workshopPdfUrlEn` and `workshopPdfUrlDe`
* Set the respective values to `https://terrestris.github.io/geoserver-introduction-ws/geoserver-introduction-ws_en.pdf`/`https://terrestris.github.io/geoserver-introduction-ws/geoserver-introduction-ws_de.pdf`
* Adjust the variable names in the text

Please make sure that you have completed the steps of the 
[preparatory work and general information](environment/README.md)-chapter to ensure a smooth and frictionless process.

The workshop is based on a number of modules. In each module you will solve a series of tasks to achieve a specific goal. Each module will expand and strengthen your knowledge about GeoServer.

The following modules will be part of the workshop:

* [Preparatory work and general information:](environment/README.md)
Basic knowledge about the workshop environment (OSGeoLive, paths, URLs, Credentials)

* [Basic knowledge about GeoServer:](basics/README.md)
General information about GeoServer

* [Administration interface:](ui/README.md)
Configuration of GeoServer via the web interface

* [Data publishing:](data/README.md)
Publish geodata from different data sources

* [Layer groups:](group/README.md)
Publish a layer group

* [Styling:](styling/README.md)
Style a layer with SLD (Styled Layer Descriptor)

# Authors

{% for author in book.authors %}
  - {{ author.name }} ([{{ author.mail }}](mailto:{{ author.mail }}))

> {% endfor %} <!--alle Autoren aktuell?-->

Author: Please add yourself to the list of authors.
